src/incubator/TextField/index.tsx

```diff
@@ -125,6 +125,7 @@ const TextField = (props: InternalTextFieldProps) => {
           <View flexG /* flex row */>
             {floatingPlaceholder && (
               <FloatingPlaceholder
+                defaultValue={others.defaultValue}
```

Comment:
Why not pass the input's 'value'? Why do we need a separate prop?

Reply:
This is a prop from RN which we inherit and the user that opened the bug is using